auto/make

```diff
@@ -430,6 +430,12 @@ cat << END > Makefile
 
 include $NXT_MAKEFILE
 
+PYTEST := $NXT_PYTEST_NAME
+
+.PHONY: check
+check: unitd
+	@\$(PYTEST)
```

Comment:
Don't hide the command (we're showing most commands in the Makefile, with few exceptions such as rmdir(1)).

Reply:
Sure, OK